velero-plugin-for-aws/object_store.go

```diff
@@ -52,6 +52,7 @@ const (
 	serverSideEncryptionKey  = "serverSideEncryption"
 	insecureSkipTLSVerifyKey = "insecureSkipTLSVerify"
 	caCertKey                = "caCert"
+	enableSharedConfigKey    = "enableSharedConfig"
```

Comment:
Could you update the [BSL docs](https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/main/backupstoragelocation.md) to include details about this field? Although it will enabled for the VSL case in this change, it won't have any effect until we can specify the per-VSL credentials. We can update the docs when we make that change.